test/WebJobs.Script.Tests/Middleware/HttpThrottleMiddlewareTests.cs

```diff
@@ -108,10 +131,11 @@ public async Task Invoke_NoThrottle_DispatchesDirectly()
             Assert.Equal(HttpStatusCode.Accepted, (HttpStatusCode)httpContext.Response.StatusCode);
         }
 
-        [Fact]
-        public async Task Invoke_MaxParallelism_RequestsAreThrottled()
+        [Theory]
+        [InlineData(3)]
+        [InlineData(1)]
```

Comment:
Verfied serial invocation with this fix. Updating this test to verify DataBlock behavior with max concurrency set to 1